### DISCUSSION TITLE
Other architectures?

### DISCUSSION THREAD

**Discussion by assistance-informatique** [taviso/123elf#84] [General]
Why only in i386 architecture?
I'm working on a raspberry pi 4 with Fedora 34 in aarch64 architecture...
Is it possible to make a lotus 1-2-3 version for the aarch64 architecture?
Bravo for this work and thank you in advance for your answer.
Pierre Bucheli.

**Comment by taviso** [comment]
I wish it was possible, but we're using the original code from UNIX.

It would be nice if we could slowly replace the original code with portable code, but that's a long way off.

**Reply by assistance-informatique** [reply]
So hopefully... In any case, thank you for your answer... Pierre.
